docs/reference/snapshot-restore/index.asciidoc

```diff
@@ -112,36 +112,7 @@ any restored indices must be compatible.
 [[snapshot-cluster-compatibility]]
 === Snapshot version compatibility
 
-[cols="6"]
-|===
-| 5+^h| Cluster version
-^h| Snapshot version ^| 2.x ^| 5.x ^| 6.x ^| 7.x ^| 8.x
-^| *1.x* -> ^|{yes-icon} ^|{no-icon}  ^|{no-icon}  ^|{no-icon}  ^|{no-icon}
-^| *2.x* -> ^|{yes-icon} ^|{yes-icon} ^|{no-icon}  ^|{no-icon}  ^|{no-icon}
-^| *5.x* -> ^|{no-icon}  ^|{yes-icon} ^|{yes-icon} ^|{no-icon}  ^|{no-icon}
-^| *6.x* -> ^|{no-icon}  ^|{no-icon}  ^|{yes-icon} ^|{yes-icon} ^|{no-icon}
-^| *7.x* -> ^|{no-icon}  ^|{no-icon}  ^|{no-icon}  ^|{yes-icon} ^|{yes-icon}
-|===
-
-You can't restore a snapshot to an earlier version of {es}. For example, you
-can't restore a snapshot taken in 7.6.0 to a cluster running 7.5.0.
-
-ifeval::["{release-state}"!="released"]
-[[snapshot-prerelease-build-compatibility]]
-NOTE: This documentation is for {es} version {version}, which is not yet
-released. The compatibility table above applies only to snapshots taken in a
-released version of {es}. If you're testing a pre-release build of {es} then you
-can still restore snapshots taken in earlier released builds as permitted by
-this compatibility table. You can also take snapshots using your pre-release
-build, and restore them using the same build. However once a pre-release build
-of {es} has written to a snapshot repository you must not use the same
-repository with other builds of {es}, even if the builds have the same version.
-Different pre-release builds of {es} may use different and incompatible
-repository layouts. If the repository layout is incompatible with the {es} build
-in use then taking and restoring snapshots may result in errors or may appear to
-succeed having silently lost some data. You should discard your repository
-before using a different build.
-endif::[]
+include::snapshot-vers-compat.asciidoc[]
 
 [discrete]
 [[snapshot-index-compatibility]]
```

Comment:
Oh also could we refresh the numbers in this section to talk about 6/7/8 rather than 5/6/7?

Reply:
Thanks. The example now uses 6.8, 7.16, and 8.0. I'll dial back the major version during the applicable 7.x backports.